code/_core/obj/item/clothing/back/storage/bluespace/_bluespace.dm

```diff
@@ -7,7 +7,7 @@
 
 
 	dynamic_inventory_count = MAX_INVENTORY_X*4
-	container_max_size = SIZE_5
+	container_max_size = SIZE_5*3
```

Comment:
You should be using the size defines for this (SIZE_5,SIZE_6,SIZE_7) instead of directly multiplying it.

Reply:
i tried this, it threw an error

Reply:
What error did it throw?

Reply:
it just wouldn't compile. i don't know exactly